guidance/models/_azureai_studio.py

```diff
@@ -106,25 +93,16 @@ def _generator(self, prompt, temperature: float):
         # TODO: Move this somewhere more general for all chat models?
         if messages == []:
             raise ValueError(
-                f"The model is a Chat-based model and requires role tags in the prompt! \
+                f"AzureAIStudio currently only supports chat-based models and requires role tags in the prompt! \
```

Comment:
If we're also supporting completions, we should update this error message to tell users about the `endpoint_type` kwarg (if they intended to use a completion only model and didn't set the new kwarg)

Reply:
Have tweaked this error message, and attempted to add a matching one for when the completion endpoint detects role tags.